app/components/dashboard/card/tooltip_component.rb

```diff
@@ -11,10 +11,17 @@ def render?
 
   def tooltip_content
     return definitions.to_s if !definitions.is_a? Hash
-
     definitions.collect do |name, description|
-      content_tag :p, class: "mb-4px" do
-        content_tag(:strong, name) << ": #{description}"
+      if name == "divider"
+        tag :hr, class: "bg-white o-65 mt-4px mb-4px"
+      else
+        content_tag :p, class: "mb-4px" do
+          if name == "note"
```

Comment:
We should not use the name here as the condition for italicising. This couples styling with the content of the tooltip.

Reply:
This was the only way I could think to do this - I am open for suggestion on better ways to add styling.

Reply:
There's another way to fix this issue. We can revert all the changes to the tooltip component and tooltip partial. But apply the styles in the `en.yml` file. This will keep the tooltip very generic.